Assets/Scripts/Singletons/NotebookManager.cs

```diff
@@ -0,0 +1,103 @@
+using UnityEngine;
+
+namespace ProjectWendigo
+{
+    public class NotebookManager : MonoBehaviour
+    {
+        [SerializeField] private GameObject _notebook;
+        private NotebookController _notebookController;
+        private NotebookNavigation _notebookNavigation;
+        private NotebookSection _journal;
+        private NotebookSection _maps;
+        private NotebookSection archive;
```

Comment:
Who approved this? Absolutely unacceptable.

Change `archive` to `_archive` or perish.